### DISCUSSION TITLE
part-ii/2019-132/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#27] [Question comments]
Part II 2019 Riemann Surfaces: Paper 2, Section II, F 

https://questions.tripos.org/part-ii/2019-132/

**Comment by artie2000** [comment]
Anyone got a hint for (b)? I think it's singular everywhere but no clue how to show

**Reply by artie2000** [reply]
Update: I've done it, the singular set is actually $\{-1\}$
